doc/content/devices/models/mkr1310/temp.md

```diff
@@ -0,0 +1,205 @@
+  if (!modem.begin(EU868)) {
+    Serial.println("Failed to start module");
+    while (1) {}
+  };
+  
+  dht.begin();
+
+  int connected = modem.joinOTAA(appEui, appKey);
+  if (!connected) {
+    Serial.println("Something went wrong; are you indoor? Move near a window and retry");
+    while (1) {}
+  }
+}
+
+void loop() {
+  int t = dht.readTemperature() * 100;
+  int h = dht.readHumidity() * 100;
+
+  byte payload[4];
+  payload[0] = highByte(t);
+  payload[1] = lowByte(t);
+  payload[2] = highByte(h);
+  payload[3] = lowByte(h);
+
+  Serial.println("Temperature: ");
+  Serial.println(dht.readTemperature());
+  Serial.println("Humidity: ");
+  Serial.println(dht.readHumidity());
+
+  modem.beginPacket();
+  modem.write(payload, sizeof(payload));
+  modem.endPacket(true);
+  delay(100000);
+}
+```
+## Uploading the code
+
+Now to upload the code to the MKR WAN 1310. First plug the Arduino into your computer. In the IDE, click on **Select Board** at the top, and select your **Arduino MKR WAN 1310**.
+
+{{< figure src="../select-board.png" alt="Selecting Board" >}}
+
+Once that is done you can upload the file to the board (Using the **->** button). Once the code is uploaded successfully, open the Serial Monitor. You can open it by clicking the **icon in the top-right corner**.
+
+{{< figure src="../serial-monitor.png" alt="Serial Monitor button highlighted" >}}
+
+Here you can check if the temperature and humidity values are showing up correctly.
+
+{{< figure src="../serial-monitor-temp.png" alt="Serial Monitor" width="55%" >}}
+
+## {{% tts %}}
+
+Now that the device is working go back to {{% tts %}} and set up the payload formatter. A payload formatter will turn the hexadecimal string received into readable text. To do this, in your application on {{% tts %}}, click on **Payload formatters**.
+
+{{< figure src="../stack-pf.png" alt="payload formatter highlighted" width="70%" >}}
+
+Make sure that **Custom Javascript formatter** is selected here from the dropdown.
+
+{{< figure src="../custom-js.png" alt="custom javascript selected in dropdown" width="70%"   >}}
+
+Now in the **Formatter code** field, replace it with this formatter:
+
+```js
+function decodeUplink(input) {
+  // Read the temperature and humidity from the payload
+  var t = (input.bytes[0] << 8 | input.bytes[1]) / 100;
+  var h = (input.bytes[2] << 8 | input.bytes[3]) / 100; 
+  
+  return {
+    data: {
+      temperature: t,
+      humidity: h
+    }
+  };
+}
+```
+
+Now if you go back to the **Live Data** tab you'll see the temperature and humidity show up!
+
+{{< figure src="../live-data.png" alt="Live data tab">}}
+
+This concludes the example project. Now go create your own project!
+
+## Troubleshoot
+
+**My DHT sensor does not display any data**
+- Make sure that you have wired it up properly.
+- If you are using a bare DHT11, make sure that it has a pull up resistor between Vcc and Signal. Consult google for more info.
```

Comment:
Again, too little info for a troubleshoot section. You can instead add a sentence like "If Arduino IDE Serial Monitor doesn't show DHT sensor data, make sure to double-check wiring." to the part where you explain these values showing up in Serial Monitor. And the second point about pull up resistor - it would be good if you can find some example wiring guide and link it in the beginning of this article together with that note about using the resistor. "Consult google" is a bit pretentious - of course everyone trying to do this and having issues will consult Google, and it doesn't add any real value to the article so pls remove